src/window_help.h

```diff
@@ -41,7 +41,7 @@ class Window_Help : public Window_Base {
 	 * @param text text to show.
 	 * @param align text alignment.
```

Comment:
you forgot to add the new param to the doc comment.
Otherwise I think this patch is fine.

Reply:
Fixed.